docs/Documentation/Features/Npcs.md

```diff
@@ -22,9 +22,58 @@ BetonQuest provides Integrations for the following Npc plugins:
 ## Referring an Npc
 
 Npcs are defined in the `npcs` section.
-For exact instruction definition see the respective integration.
+### Example
```

Comment:
It's not the style of the documentation to have an heading called example. In my opinion just remove the heading